src/Database.php

```diff
@@ -340,6 +341,33 @@ public function listCollections(array $options = [])
         return $operation->execute($server);
     }
 
+    /**
+     * Modifies a collection or view.
+     *
+     * @see ModifyCollection::__construct() for supported options
+     * @param string $collectionOrView   Collection or view to modify
+     * @param string $collModOptionValue Options and values for collMod
+     * @param array  $options            Command options
+     * @throws InvalidArgumentException for parameter/option parsing errors
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function modifyCollection($collectionOrView, $collModOptionValue, array $options = [])
```

Comment:
Also, let's type-hint `$collectionOptions` as an array. Earlier I mentioned that it could be an array or object, since it's a document, but it's easier if we just stick to an array. It's unlikely that any users are going going to pass in an object here.

Expecting an array is also consistent with the CreateIndexes command, as we expect each index specification (technically a document) to be a PHP array. CreateCollection also takes collection options as an array, but that's somewhat special since it's mixed in with the operation options (e.g. writeConcern, typeMap).